docs/docs/deploy/deploy-rasa.mdx

```diff
@@ -161,20 +164,143 @@ You can find an example that shows how to download data files from a git reposit
 
 ### 4. Deploy Rasa Assistant
 
+  ```bash
```

Comment:
This looks a little out of place. Should this be moved under "Run the following commands"?